frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -127,7 +129,22 @@ private void handleListModels(ChannelHandlerContext ctx, FullHttpRequest req) {
     }
 
     private void handlePing(ChannelHandlerContext ctx) {
-        NettyUtils.sendJsonResponse(ctx, new StatusResponse("healthy"));
+        Map<String, String> status = new HashMap<>();
+        Map<String, Model> models = ModelManager.getInstance().getModels();
+        HttpResponseStatus ret = HttpResponseStatus.OK;
+        for (Map.Entry<String, Model> m : models.entrySet()) {
+            int numScaled = m.getValue().getMinWorkers();
+            int numWorking = m.getValue().getCurrentWorkers().get();
+            String s = numScaled + "/" + numWorking;
+            if (numScaled > 0) {
```

Comment:
This might be problematic if the model thread is restarting.

Reply:
That should be OK. The idea here is that we display how many they assigned and how many they have running and ready to serve. This is not used for scaling, customers will be asked to use the "list" command to scale and that will have all the state info.